Dockerfile

```diff
@@ -0,0 +1,10 @@
+ARG base=ubuntu:18.04
```

Comment:
While this seems the obvious choice is ubuntu the current best practice for docker images?

Reply:
Normally I would go for Alpine but this doesn't work, without additional build-tools. And when you install build-tools the container is pretty much the size of the normal ubuntu image..